src/vector/geometry.rs

```diff
@@ -170,11 +170,7 @@ impl Geometry {
         // Note: C API makes no statements about this possibly returning null.
         // So we don't have to result wrap this,
         let c_str = unsafe { gdal_sys::OGR_G_GetGeometryName(self.c_geometry()) };
```

Comment:
This is essentially a static string, right?

Reply:
Apparently so, it returns a static string.